File: OneToOneCallSample/app/src/main/java/com/stringee/apptoappcallsample/service/RejectCallReceiver.java
package com.stringee.apptoappcallsample.service;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;

import com.stringee.apptoappcallsample.common.Constant;
import com.stringee.apptoappcallsample.common.NotificationUtils;
import com.stringee.apptoappcallsample.manager.CallManager;

public class RejectCallReceiver extends BroadcastReceiver {

    @Override
    public void onReceive(Context context, Intent intent) {
        NotificationUtils.getInstance(context).cancelNotification(Constant.INCOMING_CALL_ID);
        CallManager.getInstance(context).endCall(false);
    }
}

File: Kotlin-OneToOneCallSample/app/src/main/java/com/stringee/kotlin_onetoonecallsample/service/RejectCallReceiver.kt
package com.stringee.kotlin_onetoonecallsample.service

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent
import com.stringee.kotlin_onetoonecallsample.common.Constant
import com.stringee.kotlin_onetoonecallsample.common.NotificationUtils
import com.stringee.kotlin_onetoonecallsample.manager.CallManager

class RejectCallReceiver : BroadcastReceiver() {
    override fun onReceive(context: Context, intent: Intent?) {
        NotificationUtils.getInstance(context).cancelNotification(Constant.INCOMING_CALL_ID)
        CallManager.getInstance(context).endCall(false)
    }
}